parlai/agents/transformer/modules.py

```diff
@@ -727,10 +733,10 @@ def forward_embedding(
         """
         tensor = self.embeddings(input)
         if self.embeddings_scale:
-            tensor = tensor * np.sqrt(self.dim)
+            tensor = tensor * math.sqrt(self.dim)
         if self.variant == 'xlm':
             tensor = self.norm_embeddings(tensor)
-        if positions.max().item() > self.n_positions:
+        if not torch.jit.is_scripting() and positions.max().item() > self.n_positions:
```

Comment:
why is this not supported by jit?

Reply:
Yes, looks like this is something else that is supported by tracing, so reverting this